server/src/main/kotlin/pl/kubiczak/test/spring/integration/demo/employees/exposed/EmployeeTable.kt

```diff
@@ -0,0 +1,14 @@
+package pl.kubiczak.test.spring.integration.demo.employees.exposed
+
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.sql.Column
+
+object EmployeeTable : IdTable<Long>(name = "employees") {
```

Comment:
Cannot use `LongIdTable` because there is a need to provide custom sequence name.